cilium/cmd/bpf_egress_lookup.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2021 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"net"
+	"os"
+
+	"github.com/cilium/cilium/pkg/bpf"
+	"github.com/cilium/cilium/pkg/common"
+	"github.com/cilium/cilium/pkg/maps/egressmap"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	egressLookupUsage = "Lookup egress entries using source and destination IPs.\n"
+)
+
+var bpfEgressLookupCmd = &cobra.Command{
+	Args:  cobra.ExactArgs(2),
+	Use:   "lookup",
```

Comment:
I would rename this subcommand to `get` to match the ipcache map. We can have `lookup` as an alias.